config/uplink.yaml

```
@@ -13,9 +13,13 @@ ingress:
     - secretName: uplink-tls
       hosts:
         - uplink.jenkins.io
-replicaCount: 5
 image:
   # repo & '@sha256' suffix here as we're retrieving a docker digest from updatecli
   repository: jenkinsciinfra/uplink@sha256
   tag: c7ecad0e5e24bc05ac722418faf844d6a0286620c6316ff712459ecfcc3ee7ce
   pullPolicy: IfNotPresent
+
+replicaCount: 5
+
+nodeSelector:
+  agentpool: arm64small
```

```suggestion
  agentpool: x86medium
```

We haven't evaluated if an ARM image is available for this app